### ISSUE TITLE
Merged file appears to be invalid

### ISSUE THREAD

**Issue by peteryates** [DFE-Digital/presentation-merger#4]
**Describe the bug**

When testing via [the Ruby integration](https://github.com/DFE-Digital/curriculum-materials/pull/113) the merged file appears to be invalid. If I unzip `merge.odp` the contents look _ok_ (the files are laid out in the correct manner), so I'm guessing this is a minor bug or an incompatibility between various versions of the ODF standard, or different implementations across office suites.

```
[5] pry(main)> pm = PresentationMerger.new('/home/peter/presentation-test/merge.odp')
=> #<PresentationMerger:0x0000000005de1ca0
 @closed=false,
 @file_name="/home/peter/presentation-test/merge.odp",
 @files=[],
 @output_stream=#<File:/home/peter/presentation-test/merge.odp>>

[6] pry(main)> pm << File.new('/home/peter/presentation-test/one.odp')
=> [#<File:/home/peter/presentation-test/one.odp>]

[7] pry(main)> pm.close
=> "/home/peter/presentation-test/merge.odp"
```

**Expected behaviour**

I'd expect the resulting file to open in LibreOffice.

**Screenshots**

![Screenshot from 2020-03-06 09-45-24](https://user-images.githubusercontent.com/128088/76073284-4756f800-5f91-11ea-9362-03d5f437b135.png)

**Additional context**

Files attached.

LibreOffice info:
Version: 6.3.4.2.0+
Build ID: 6.3.4.2-2.fc31

Files:

Note, `merge.odp` is the invalid file, `merge-original.odp` is how it looked before the test.

[merge-test.zip](https://github.com/DFE-Digital/presentation-merger/files/4297882/merge-test.zip)

**Comment by rjlynch** [comment]
Here's the stack trace when running the cli tool directly, also worth noting it fails with a 0 exit code
![Screenshot 2020-03-06 at 10 48 21](https://user-images.githubusercontent.com/9936028/76077176-20e88b00-5f98-11ea-8b82-815644aa750e.png)

```
 => DEBUG=presentation-merger node_modules/.bin/presentation-merger pres1.odp pres2.odp -o merged.odp
  presentation-merger Dependencies loaded 594.639531ms +0ms
  presentation-merger Program loaded 5.437747ms +8ms
Error: Error parsing XML
    at Object.parse (/Users/richardlynch/Development/dfe/curriculum-materials-project/curriculum-materials/node_modules/@prettier/plugin-xml/src/parse.js:7:11)
    at Object.parse (/Users/richardlynch/Development/dfe/curriculum-materials-project/curriculum-materials/node_modules/prettier/index.js:9739:19)
    at coreFormat (/Users/richardlynch/Development/dfe/curriculum-materials-project/curriculum-materials/node_modules/prettier/index.js:13252:23)
    at format (/Users/richardlynch/Development/dfe/curriculum-materials-project/curriculum-materials/node_modules/prettier/index.js:13510:73)
    at formatWithCursor (/Users/richardlynch/Development/dfe/curriculum-materials-project/curriculum-materials/node_modules/prettier/index.js:13526:12)
    at /Users/richardlynch/Development/dfe/curriculum-materials-project/curriculum-materials/node_modules/prettier/index.js:44207:15
    at format (/Users/richardlynch/Development/dfe/curriculum-materials-project/curriculum-materials/node_modules/prettier/index.js:44226:12)
    at Document.toFormattedXML (/Users/richardlynch/Development/dfe/curriculum-materials-project/curriculum-materials/node_modules/@dfe-digital/presentation-merger/lib/Document.js:133:35)
    at Document.pipe (/Users/richardlynch/Development/dfe/curriculum-materials-project/curriculum-materials/node_modules/@dfe-digital/presentation-merger/lib/Document.js:87:33)
    at mergeFile (/Users/richardlynch/Development/dfe/curriculum-materials-project/curriculum-materials/node_modules/@dfe-digital/presentation-merger/lib/index.js:19:7)
  presentation-merger Merged files in 457.670634ms +459ms
  presentation-merger Program total time 61.766771ms +0ms
```

Here's the presentations I tried merging along with the result `merged.odp`
[samples.zip](https://github.com/DFE-Digital/presentation-merger/files/4297922/samples.zip)

**Comment by kylewelsby** [comment]
It is discovered that this issue is caused by the `style.xml` `style:name` clashing between presentations. 

#### Suggested resolution

Do an initial pass on all `style:name` keys prefixing with a unique presentation ID/number
